platform-ajax/src/main/java/com/softicar/platform/ajax/testing/selenium/engine/level/low/interfaces/IAjaxSeleniumLowLevelTestEngineMethods.java

```diff
@@ -756,4 +780,9 @@ default void simulateInput(IDomNode target) {
 
 		getTestEngine().getEventSimulator().simulateInput(target);
 	}
+
+	default void simulateWheel(IDomNode node, int deltaY) {
```

Comment:
what is the unit of `deltaY`?

Reply:
we might add that as Javadoc, what you think?

Reply:
did that. also found out that px is the default.